### DISCUSSION TITLE
multi select component bug ?

### DISCUSSION THREAD

**Discussion by KreatorXoX** [JedWatson/react-select#5586] [Q&A] [answered]
I am having difficulty using react-select with the isMulti prop. Even though it is meant to be a multi-select component, when I choose an option, react-select indicates that there are no other options available.

 example:
https://codesandbox.io/embed/react-typescript-forked-67hdk7?fontsize=14&hidenavigation=1&theme=dark

I know that it has something to do with js map function but i can't figure out on my own.

**Comment by manjushsh** [comment, marked as answer]
Hi @KreatorXoX
In mock data, you have same values in price key. That is the reason why you see No options once you select a option.
Just updating mock data should fix this. Or use `id` in value key instead of `price`
[Working sandbox link](https://codesandbox.io/s/react-typescript-forked-9moiqm?file=/src/App.tsx)

![image](https://user-images.githubusercontent.com/94426452/226241011-b450b268-ed79-43c7-bfd9-8707ad6744ce.png)

**Reply by KreatorXoX** [reply]
Thanks for the answer. Appreciated <3
